koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -146,3 +146,32 @@ def get_last_seen_manifest_ids(self, bill_date):
             if not processed:
                 assembly_ids.append(manifest.assembly_id)
         return assembly_ids
+
+    def delete_cost_usage_reports_older_than(self, provider_type, billing_period_start_datetime):
+        """
+        Deletes Cost usage Reports older than billing_period_start_datetime
+
+        Args:
+            provider_type   (String) the provider type to delete
+            billing_period_start_datetime (datetime.datetime) delete all manifests older than this date, exclusive.
+        """
+        delete_count = CostUsageReportManifest.objects.filter(
+            provider__type=provider_type, billing_period_start_datetime__lt=billing_period_start_datetime
+        ).delete()
+        LOG.info(
+            "Removed %s CostUsageReportManifest(s) for provider type %s that had a billing period start date before %s",
+            delete_count,
+            provider_type,
+            billing_period_start_datetime,
+        )
+
+    def delete_provider_cost_usage_reports_older_than(self, provider_uuid, billing_period_start_datetime):
+        delete_count = CostUsageReportManifest.objects.filter(
```

Comment:
if you don't do the optional suggestion above you will need a doc string here.

Reply:
added